calc/.eslintrc

```diff
@@ -0,0 +1,343 @@
+{
+    "root": true,
+    "parserOptions": {
+        "ecmaVersion": 3,
+        "sourceType": "script"
+    },
+    "env": {
+        "shared-node-browser": true
+    },
+    "globals": {
+        "exports": false,
+        "buildDescription": false,
+        "checkAirLock": false,
+        "checkDownload": false,
+        "checkIntimidate": false,
+        "checkKlutz": false,
+        "checkForecast": false,
+        "getMultiAttack": false,
+        "Field": false,
+        "NATURES": false,
+        "Pokemon": false,
+        "getFinalSpeed": false,
+        "getItemBoostType": false,
+        "getNaturalGift": false,
+        "getMoveEffectiveness": false,
+        "getTechnoBlast": false,
+        "getFlingPower": false,
+        "isGroundedForCalc": false,
+        "countBoosts": false,
+        "displayStat": false,
+        "toSmogonStat": false,
+        "getModifiedStat": false,
+        "getBerryResistType": false,
+        "megaStones": false,
+        "STATS": false,
+        "seedBoostedStat": false,
+        "mega_Stones": false,
+        "CALCULATE_DAMAGE_RBY": false,
+        "CALCULATE_DAMAGE_GSC": false,
+        "CALCULATE_DAMAGE_ADV": false,
+        "CALCULATE_DAMAGE_DPP": false,
+        "CALCULATE_DAMAGE_BW": false,
+        "calculateDamge": false,
+        "ZMOVES_TYPING": false,
+        "calcStatRBYFromDV": false,
+        "calcStatADV": false,
+        "SPECIES": false,
+        "MOVES": false,
+        "ABILITIES": false,
+        "ITEMS": false,
+        "TYPE_CHART": false,
+        "getKOChanceText": false,
+        "HP": false,
+        "AT": false,
+        "DF": false,
+        "SP": false,
+        "SD": false,
+        "SL": false,
+        "SA": false,
+        "Move": false,
+        "extend": false
+    },
+    "extends": "eslint:recommended",
+    "rules": {
+        "no-cond-assign": [
+            "error",
+            "except-parens"
+        ],
+        "no-console": "off",
+        "no-constant-condition": "off",
+        "no-control-regex": "off",
+        "no-empty": [
+            "error",
+            {
+                "allowEmptyCatch": true
+            }
+        ],
+        "no-inner-declarations": [
+            "error",
+            "functions"
+        ],
+        "no-redeclare": "off",
+        "valid-jsdoc": "off",
+        "array-callback-return": "error",
+        "complexity": "off",
+        "consistent-return": "off",
+        "default-case": "off",
+        "dot-location": [
+            "error",
+            "property"
+        ],
+        "dot-notation": "off",
+        "eqeqeq": "off",
+        "guard-for-in": "off",
+        "no-caller": "error",
+        "no-case-declarations": "off",
+        "no-div-regex": "error",
+        "no-else-return": "off",
+        "no-labels": [
+            "error",
+            {
+                "allowLoop": true,
+                "allowSwitch": true
+            }
+        ],
```

Comment:
We should probably go through these and remove any un-needed ones as this was copied from PS

Reply:
Well, this is only temporary anyway. Once I've converted to TS I will be deleting it in favor of the `tslint.json` (and automatic code formatting).

Reply:
I actually realized I could just remove `"root": true` and inherit almost all of this. So while its still temporary, its now short as well.